packages/core/src/utils/utils.ts

```diff
@@ -394,4 +394,4 @@ export const colorScheme = ({ colorScheme, ...props }) => {
     border-color: ${foreground};
     color: ${foreground};
   `
-}
\ No newline at end of file
+}
```

Comment:
The newline here and in `packages/icons/src/index.js`  arenot on `main` at this moment, but seem to be a sensible `prettier` change, right?